plugins/ImageGraph/StaticGraph.php

```diff
@@ -386,3 +391,15 @@ function formatYAxis($value)
 {
     return NumberFormatter::getInstance()->format($value);
 }
+
+/**
+ * Global format method - non-unifont
+ *
+ * required to format y axis values using CpChart internal format callbacks
+ * @param $value
+ * @return mixed
+ */
+function formatYAxisNonUnifont($value)
+{
+    return str_replace("\xE2\x80\xAF", " ", NumberFormatter::getInstance()->format($value));
```

Comment:
```suggestion
    // replace narrow no-break spaces with classic space as some fonts may not support it
    return str_replace("\xE2\x80\xAF", " ", NumberFormatter::getInstance()->format($value));
```

Reply:
Just a thought — I couldn't think of any case where we should prevent line-break in the y-axis, but would the font still support classic NBSP instead? Or do we need to go with a classic space only?

Reply:
I suppose if the thousand separator for 1 234  had a line break between the 1 and the 234 then it would be easy to misread the data as 234, so it would probably be better to avoid line breaks if possible.

I tested the non-unifont default of `tahoma.ttf` using a `\xC2\xA0` (NBSP) character instead of a normal space and it seemed to work well, so I've committed that change and the suggested comment :+1: